public/angular/sass/app.scss

```diff
@@ -50,6 +50,10 @@ a {
   }
 }
 
+.navbar-nav > li > ul {
+  background-color: darken($navbar-green, 10%);
+}
```

Comment:
The color looks good. Can we make the menu items a bit roomier? Perhaps same font size and height as in nav bar?